securedrop/template_filters.py

```diff
@@ -9,7 +10,7 @@ def datetimeformat(dt, fmt=None, relative=False):
     if relative:
         time_difference = _relative_timestamp(dt)
         if time_difference:
-            return '{} ago'.format(time_difference)
+            return gettext('{time} ago').format(time=time_difference)
```

Comment:
A problem I see with this is that some languages might not say "$time ago" and that the values may be ordered a different way. This should duplicate the "ago" in all the strings. Also, this will help the translators.

Reply:
It is indeed the case in French (il y a 2 jours => 2 days ago) but it is consistent and this factorization did not cause problem. I propose we wait until someone complains that it is causing problem (arabic comes to mind as the rules are significantly different).

Reply:
and Dutch + Norwegian are also ok with it it seems.

Reply:
In German it's "vor 2 Tagen" for "2 days ago," so that's already one.